functions/common.ts

```diff
@@ -0,0 +1,19 @@
+interface Assignee {
+  name: string;
+}
+
+interface UnifiedIssue {
+  title: string;
+  assignees: Assignee[];
+  author: Assignee;
+  body: string;
+  category: string;
+  statusChangeTime: Date;
+  createdAt: Date;
+  comments: string[];
```

Comment:
```suggestion
  comments: number;
```
According to the excel sheet Github doesn't give the actual comments, only the amount. Correct me if I'm wrong.